likecoin/js/admin/likecoin_metabox.js

```diff
@@ -12,7 +17,30 @@ async function onRefreshPublishStatus(e) {
       xhr.setRequestHeader('X-WP-Nonce', wpApiSettings.nonce);
     },
   });
-  const { matters, ipfs } = res;
+  const { matters, ipfs, arweave } = res;
+  const wordpressPublished = res.wordpress_published;
+  if (iscnHash && iscnId) {
+    const iscnIdString = encodeURIComponent(iscnId);
+    lcPostInfo.mainStatusTitle = 'LIVE on #DePub';
+    mainStatusField.textContent = '';
+    mainStatusField.innerHTML = `<h3>${lcPostInfo.mainStatusTitle}</h3>`;
+    arweaveTextField.innerHTML = `<a rel="noopener" target="_blank" href="https://app.like.co/view/${iscnIdString}">${iscnId}</a>`;
+  } else if (wordpressPublished === 'publish' && (arweaveAndISCNUploadStatus === 'initial' || arweaveAndISCNUploadStatus === 'failed')) {
+    lcPostInfo.mainStatusTitle = 'READY to #DePub';
+    mainStatusField.textContent = '';
+    mainStatusField.innerHTML = `<h3>${lcPostInfo.mainStatusTitle}</h3>`;
+    const uploadArweaveBtn = document.createElement('button');
+    uploadArweaveBtn.innerText = 'Submit to ISCN';
+    uploadArweaveBtn.setAttribute('id', 'lcArweaveUploadBtn');
+    uploadArweaveBtn.setAttribute('class', 'button button-primary');
+    arweaveTextField.textContent = '';
+    arweaveTextField.appendChild(uploadArweaveBtn);
```

Comment:
Please refactor these clear and append into function too
Actually https://developer.mozilla.org/en-US/docs/Web/API/Element/replaceChildren would be simpler but no ie support

Reply:
Will refactor to function, but keep appendChild to accommodate IE support